CHANGELOG.rst

```diff
@@ -1,6 +1,73 @@
 Changelog
 =========
 
+1.6.0 (2022-03-??)
+------------------
+
+**New features**
+
+* Upgraded to Django 3.2 LTS version (#1098)
+* Upgraded to zgw-consumers 0.18.0, which supports client/server certificate
+  configuration for external services. This effectively adds Mutual TLS capabilities to
```

Comment:
Is this true? I don't think this allows you to do certificate checking by the Zaken, Documenten, etc. API's themselves, right?

Reply:
it should work for connections to BAG, BRP etc. that can be defined under the "external services":

![image](https://user-images.githubusercontent.com/5518550/159059018-787b0b8a-5a41-4117-81d2-97bc7ff4b57d.png)

Reply:
Yes but the last sentence implies Zaken API etc have it as well

Reply:
I'll make sure to reword it!